table/server/underdb/hive/src/main/java/alluxio/table/under/hive/HiveDatabase.java

```diff
@@ -112,7 +113,7 @@ public String getName() {
   public List<String> getTableNames() throws IOException {
     try {
       return getHive().getAllTables(mHiveDbName);
-    } catch (MetaException e) {
+    } catch (TException  e) {
```

Comment:
This had to be upgraded to a TException because the interface actually throws TException.